client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -95,7 +106,11 @@ const RobotOverview = () => {
       <>
         {filteredBotList.map((val) => (
           // eslint-disable-next-line dot-notation
-          <RobotContainer robotId={val['_id']} robotName={val.robotName} />
+          <RobotContainer
+            userId={userId}
+            robotId={val['_id']}
```

Comment:
if possible please use the dot notation here; otherwise please use inline fix for aslant

Reply:
✅